Sources/Purchasing/StoreKit1/PaymentQueueWrapper.swift

```diff
@@ -14,12 +14,33 @@
 import Foundation
 import StoreKit
 
-final class PaymentQueueWrapper {
+protocol PaymentQueueWrapperDelegate: AnyObject, Sendable {
```

Comment:
It's a bit confusing that this is defined in the `StoreKit1` folder but we will be using it in SK2. Maybe it should be moved out of that folder?

Reply:
It does wrap in `SK1` type. When we get rid of SK1 support it'll help that it's there :)